readme.md

```diff
@@ -10,7 +10,7 @@ See Likely in action on its [homepage](http://ilyabirman.net/projects/likely/).
 
 ## Get
 
-[Download the repository code](https://github.com/ilyabirman/Likely/archive/master.zip) and move `release/likely.js` and 
+[Download the repository code](https://github.com/ilyabirman/Likely/archive/master.zip) and move `release/likely.js` and
```

Comment:
Can we avoid such diffs in commits? It’s OK to do these changes, but I’d push them as a separate commit to make the change history cleaner.